buildspecs/publish.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+arch_type=$1
+debug=$2
+
+if [[ -z "$arch_type" ]];
+  then
+    echo "No architecture type specified!"
+    echo "exiting..."
+    exit 1
+fi
+
+region="${AWS_REGION:-us-east-1}";
+echo "getting caller identity"
+callerIdentity=$(aws sts get-caller-identity);
+accountId=$(jq -r .Account <<< $callerIdentity);
+## TODO: use this once our custom alias is approved
+# ecrEndpoint="public.ecr.aws/tinystacks/";
+ecrEndpoint="public.ecr.aws/m7b0o7h1";
+echo "getting version"
+version=$(cat version);
+echo "getting name"
+appName=$(cat name);
```

Comment:
We'll need to update these.  In the lambda extensions it was just bash so we kept these two files specifically for this.  Since this repo is a node app we should pull this info from the package.json